webextensions/api/windows.json

```diff
@@ -1466,7 +1466,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤60"
+                  "version_added": "4"
```

Comment:
✅ https://github.com/chromium/chromium/commit/cf9db21597c56cbec3c87ba0304d74e465543a2d in [4.0.212.1](https://github.com/chromium/chromium/releases/tag/4.0.212.1) before 4.0.223.